packages/server/database/migrations/20221123161224-updateTier.ts

```diff
@@ -0,0 +1,181 @@
+import {R} from 'rethinkdb-ts'
+import getPg, {closePg} from '../../postgres/getPg'
+
+export const up = async function (r: R) {
+  const BATCH_SIZE = 1000
+  for (let i = 0; i < 1e5; i++) {
+    const skip = i * BATCH_SIZE
+    const res = await Promise.all([
+      r
+        .table('Organization')
+        .skip(skip)
+        .limit(BATCH_SIZE)
+        .update((row) => ({
+          tier: r.branch(
+            row('tier').eq('personal'),
+            'starter',
+            r.branch(row('tier').eq('pro'), 'team', row('tier'))
+          )
+        }))
+        .run(),
+      r
+        .table('OrganizationUser')
+        .skip(skip)
+        .limit(BATCH_SIZE)
+        .update((row) => ({
+          tier: r.branch(
+            row('tier').default(null).eq('personal'),
+            'starter',
+            r.branch(row('tier').default(null).eq('pro'), 'team', row('tier').default(null))
+          ),
+          suggestedTier: r.branch(
+            row('suggestedTier').default(null).eq('personal'),
+            'starter',
+            r.branch(
+              row('suggestedTier').default(null).eq('pro'),
+              'team',
+              row('suggestedTier').default(null)
+            )
+          )
+        }))
+        .run(),
+      r
+        .table('Invoice')
+        .skip(skip)
+        .limit(BATCH_SIZE)
+        .update((row) => ({
+          tier: r.branch(
+            row('tier').eq('personal'),
+            'starter',
+            r.branch(row('tier').eq('pro'), 'team', row('tier'))
+          )
+        }))
+        .run(),
+      r
+        .table('User')
+        .skip(skip)
+        .limit(BATCH_SIZE)
+        .update((row) => ({
+          tier: r.branch(
+            row('tier').eq('personal'),
+            'starter',
+            r.branch(row('tier').eq('pro'), 'team', row('tier'))
+          )
+        }))
+        .run(),
+      r
+        .table('Team')
+        .skip(skip)
+        .limit(BATCH_SIZE)
+        .update((row) => ({
+          tier: r.branch(
+            row('tier').eq('personal'),
+            'starter',
+            r.branch(row('tier').eq('pro'), 'team', row('tier'))
+          )
+        }))
+        .run()
+    ])
+
+    const isComplete = res.every(
+      (res) => res.skipped === 0 && res.unchanged === 0 && res.errors === 0 && res.replaced === 0
+    )
+    if (isComplete) break
+  }
+
+  const pg = getPg()
+
+  await pg.query(
+    `
+    ALTER TYPE "TierEnum" RENAME VALUE 'personal' TO 'starter';
```

Comment:
The TypeScript check is failing because the generated postgres files haven't updated the `TierEnum`. However, when I run `yarn dev` and then `yarn typecheck` locally, I don't get any errors as I can see the generated files have updated to include the new `TierEnum`.

Any ideas why the generated postgres files in this build don't include the new `TierEnum`? Do you get any TS errors when running `yarn typecheck` locally?